homeassistant/components/sensor/iaqstick.py

```diff
@@ -0,0 +1,131 @@
+"""
+Support for AppliedSensor iAQStick
+
+On Linux, a custom udev rule is needed, add a file
+/etc/udev/rules.d/99-iaqstick.rules containing
+
+SUBSYSTEM=="usb", ATTR{idVendor}=="03eb", ATTR{idProduct}=="2013", MODE="666"
+
+and run # udevadm control --reload-rules && udevadm trigger or reboot.
```

Comment:
Those details should go into the documentation. Please follow the style for the file header.